docs/en/siem/cases-overview.asciidoc

```diff
@@ -0,0 +1,6 @@
+[[cases-overview]]
+[role="xpack"]
+
+= Cases
+
+// Placeholder
```

Comment:
delete this file? not sure what this is

Reply:
@stephmilovic - it's just a placeholder. It's used in the PR for the UI docs (https://github.com/elastic/stack-docs/pull/967)